app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1030,6 +1030,12 @@ def validate(self, data):
         return data
 
     def update(self, experiment, validated_data):
+        population_percent = validated_data.get("population_percent")
+        if population_percent != experiment.population_percent and experiment.status in (
+            NimbusConstants.Status.LIVE
+        ):
```

Comment:
Oh I don't think there's any need to check individual fields here.  That won't scale well as we continue to allow different fields.  I think the only thing we care about here is if it's live and we're updating, it's dirty now.

Reply:
I was using it as a way to rule out end enrollment and end experiment updates, which are also status=LIVE. I can switch it out though, I think you're right that it doesn't make sense here 👍

Reply:
Oh good point about the ending cases, but you should be able to determine that by looking at `status` and `status_next`.  And in the ending enrollment case, in theory that should go away as a special case altogether and just fold into the DIRTY review/update flow.

Reply:
Nah, cuz we also need to account for end enrollment/end experiment rejection! There's 5 cases total to think about, and I think this is the smallest checks we can do to ensure that all of them are covered:
<img width="888" alt="image" src="https://user-images.githubusercontent.com/43795363/219106961-40be84b3-b17a-4866-b10d-82e4f5fccbc3.png">

Reply:
(don't judge my color-coded spreadsheet)

Reply:
First of all. 👏 👏 👏 👏 

Second of all, shouldn't it be (status, status_next, publish_status)`LIVE/LIVE/IDLE` always goes to `LIVE/LIVE/DIRTY`?

Reply:
In the case where an update comes in from V5?

Reply:
Like we only care about the DIRTY UPDATE case in your diagram, and it would have gone from `LIVE/NONE/IDLE` to `LIVE/NONE/IDLE` ie not touching the states at all, but we can just bump `IDLE` to `DIRTY`?

Reply:
Not necessarily, `LIVE/NONE/IDLE` can be from ending an enrollment or ending an experiment as well (see the lefthand side of Dirty Updates, End Enrollment, and End Experiment on my handy-dandy spreadsheet). All three updates can start from the same `LIVE/NONE/IDLE` values, so we need to 
1. add isPaused into the mix to rule out End Enrollment
2. check the new val for publish status (Review for both end exp and end enroll) and/or check the new val for status next (Complete for end exp)